stackrox/stackrox
# ROX-10246 add log message to deployment check if network criteria policy is evaluated

## Description

When users run `deployment check` there is currently no way to know if Network Policy violations will be raised.

Before enabling this feature, we should show a warning log message if this policy criteria is being check on deployment check.

For example something like:

```bash
$ roxctl deployment check ./path/to/deployment.yaml
Warning: Ingress/Egress Network Policy criteria will not be evaluated
<violations table>
```
## Checklist

If any of these don't apply, please comment below.

## Testing Performed

* Enable the default policy `Deployments should have at least one ingress Network Policy`
* Run `roxctl deployment check -f deploy.yaml`

Should return:

```bash
WARN:	Ingress/Egress Network Policy criteria will not be evaluated
Policy check results for deployments: [nginx-deployment-1]
(TOTAL: 4, LOW: 2, MEDIUM: 2, HIGH: 0, CRITICAL: 0)
<violations table>
```

```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: nginx-deployment-1
spec:
  selector:
    matchLabels:
      app: nginx-1
      role: backend
  replicas: 1
  template:
    metadata:
      labels:
        app: nginx-1
        role: backend
    spec:
      containers:
      - name: nginx
        image: nginx:1.14.2
        ports:
        - containerPort: 80
```
